rest/server_context.go

```diff
@@ -514,7 +514,11 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(ctx context.Context, config
 			MetadataIndexes: metadataIndexes,
 			UseXattrs:       config.UseXattrs(),
 		}
-		ctx := base.CollectionCtx(ctx, ds.GetName())
+		dsName, ok := base.AsDataStoreName(ds)
+		if !ok {
+			return false, fmt.Errorf("Could not get datastore name from %s", base.MD(ds.GetName()))
+		}
+		ctx := base.KeyspaceLogCtx(ctx, bucket.GetName(), dsName.ScopeName(), ds.GetName())
```

Comment:
`ds.GetName()` is fully qualified, resulting in this:

```
2023-03-16T12:36:23.042Z [INF] b:b1.foo.b1.foo.bar Initializing indexes with numReplicas: 0...
```

```suggestion
		ctx := base.KeyspaceLogCtx(ctx, bucket.GetName(), dsName.ScopeName(), ds.CollectionName())
```